examples/native-hybrid-app/flutter_module/.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
> [!NOTE]
JSON is a data-only format, and using `//` style comments could break some parsing tools (apparently not VSCode 😅 ).
I usually go with an attribute that is ignored and won't break the parsing or code style in editors, e.g.:
```
{
  "comment": "My comment about something"
}
```

Reply:
This is technically `jsonc`, which is literally just json with comments.